backend/environment/env.social-app-backend.template

```diff
@@ -0,0 +1,35 @@
+# Copy this file to "<project_root>/.env.social-app-backend", and then tweak values for local development
+# Values in this file will override the same-named environnent variables in `.env.common.docker` for the social-app-backend
+
+# Base URL for provider webhook endpoints
+PROVIDER_BASE_URL=https://some-provider/api/v1.0.0
+
+# Number of failures allowing in the provider webhook before the service is marked down
+WEBHOOK_FAILURE_THRESHOLD=3
+
+# Number of seconds between provider webhook retry attempts when failing
+WEBHOOK_RETRY_INTERVAL_SECONDS=10
+
+# Number of `/health` endpoint failures allowed before marking the provider webhook service down
+HEALTH_CHECK_MAX_RETRIES=4
+
+# Number of seconds to retry provider webhook `/health` endpoint when failing
+HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS=10
+
+# Minimum number of consecutive successful calls to the provider webhook
+# `/health` endpoint before it is marked up again
+HEALTH_CHECK_SUCCESS_THRESHOLD=10
+
+# Maximum amount of provider capacity this app is allowed to use (per epoch)
+#     type: 'percentage' | 'amount'
+#     value: number (may be percentage, ie '80', or absolute amount of capacity)
+CAPACITY_LIMIT='{"type":"percentage", "value":80}'
```

Comment:
Should this just be a default? Feels like we have a lot of default-able config in all of these. On the other hand I guess this is setting the default for the template app.